packages/component/src/Middleware/concatMiddleware.spec.js

```diff
@@ -0,0 +1,71 @@
+import concatMiddleware from './concatMiddleware';
+
+test('concat 2 middleware', () => {
+  const oddMultiplyByTen = () => next => value => {
+    if (value % 2) {
+      return value * 10;
+    }
+
+    return next(value);
+  };
+
+  const evenMultiplyByHundred = () => next => value => {
+    if (!(value % 2)) {
+      return value * 100;
+    }
+
+    return next(value);
+  };
+
+  const middleware = concatMiddleware(oddMultiplyByTen, evenMultiplyByHundred);
+  const work = middleware()(value => value);
+
+  expect(work(1)).toEqual(10);
+  expect(work(2)).toEqual(200);
+});
+
+test('run 2 middleware for a single work', () => {
+  const combine = () => next => value => {
+    return next(value) + next(value + 1);
+  };
+
+  const oddMultiplyByTen = () => next => value => {
```

Comment:
Can we reuse the `oddMultiplyByTen` and `evenMultiplyByHundred ` from the previous test? Seems redundant to define them twice.

Reply:
Fixed!